app/ide-desktop/lib/client/src/project-management.ts

```diff
@@ -99,6 +99,7 @@ export function importDirectory(rootPath: string): string {
         const targetDirectory = generateDirectoryName(rootPath)
         if (fsSync.existsSync(targetDirectory)) {
             const message = `Project directory already exists: ${targetDirectory}.`
+            // eslint-disable-next-line no-restricted-syntax
```

Comment:
since the eslint fixes have been merged in, this *should* be redundant now

Reply:
Done.